### DISCUSSION TITLE
package conflicts with an existing package

### DISCUSSION THREAD

**Discussion by titandino88** [ReVanced/revanced-discussions#1081] [Question] [answered]
java -jar cli.jar patch -b patches.jar -m integration.apk -o out.apk in.apk

when there's a new apk update, i use the same command to patch, there is no error with the out.apk

but when i try to install/update the apk, it gave error saying package conflicts with an existing package

apk source is always from apk pure

please advice what i did wrong

**Comment by LisoUseInAIKyrios** [comment]
Are you patching YouTube?  If patching failed then it will give that error because the package name is still the same.

Use the nodpi APK from ApkMirror.

**Reply by titandino88** [reply]
i'm patching X and also Google Photos, but gave the same error whenever there's a new update

i have to uninstall then reinstall the apk to get it to work which is troublesome as it clears all the settings

**Reply by LisoUseInAIKyrios** [reply, marked as answer]
Are you deleting any files in the patcher directory?  If the generated patcher signing key is deleted then an installation error will occur when trying to patch again, because it will generate a new (and different) signing key that no longer matches.

**Reply by titandino88** [reply]
ahh.. now i get it. i should not have cleared all the files after the patch

in other words i should only replace the in.apk and copy out the out.apk every time and leave all the files in the directory

thanks for the advice
